src/_common/sticker/placement/modal/modal.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<app-modal>
+		<div class="modal-controls">
+			<app-button @click="modal.dismiss()">
+				<translate>Close</translate>
+			</app-button>
+		</div>
+		<div class="modal-header">
+			<h2 class="modal-title">
+				<translate>Place Sticker</translate>
+			</h2>
+		</div>
+		<div class="modal-body">
+			<div class="help-block">
+				<translate
+					>Position the sticker in the outlined target area below, then press "Place".</translate
+				>
+			</div>
+
+			<div class="-mount">
```

Comment:
Make an overlay that will cover the "mount" content so that it doesn't get any pointer events. Also use some sort of darkening with opacity to dim it